packages/x-tree-view/src/utils/cache.ts

```diff
@@ -4,7 +4,7 @@ type DataSourceCacheDefaultConfig = {
   /**
    * Time To Live for each cache entry in milliseconds.
    * After this time the cache entry will become stale and the next query will result in cache miss.
-   * @default 300000 (5 minutes)
+   * @default 300_000 (5 minutes)
```

Comment:
Just saw those while looking for the word "cache" and it's more readable with the separator imo